app/models/work.rb

```diff
@@ -126,6 +126,8 @@ def find_by_ark(ark)
       work.validate_ids
     elsif draft?
       work.valid_to_draft
+      # There are cases where, before the README is attached, invalid metadata may be provided
```

Comment:
I think this new set of code will ignore errors in valid_to_draft.  Could these checks be added there instead?